dpdk.spec

```diff
@@ -147,11 +147,12 @@ as L2 and L3 forwarding.
 %endif
 
 %prep
-%autosetup -n dpdk-stable-%{?commit0:%{commit0}}%{!?commit0:%{ver}} -p1
+%autosetup -n dpdk-%{?commit0:%{commit0}}%{!?commit0:%{ver}} -p1
 
 %build
+pip install pyelftools
```

Comment:
We cannot use `pip` inside RPM builds.  Instead you should add a `BuildRequires: python3-pyelftools` to get that RPM installed and provide _pyelftools_.